package.json

```diff
@@ -31,14 +31,15 @@
     "url": "sindresorhus.com"
   },
   "scripts": {
-    "test": "node test.js"
+    "test": "xo && ava"
   },
   "dependencies": {
     "number-is-nan": "^1.0.0"
   },
   "devDependencies": {
     "array-equal": "^1.0.0",
-    "ava": "^0.0.4"
+    "ava": "*",
+    "xo": "*"
```

Comment:
@mshenfield I considered it, but I just have too many modules and don't want to have to bump them all the time. They're my modules anyways so I can update them when I make breaking changes, which will be very rare going forward.